### ISSUE TITLE
Distributed firewall rule with the object urn:vcloud:firewallGroup:internal for all internal vDC Group network traffic

### ISSUE THREAD

**Issue by odraghi** [vmware/terraform-provider-vcd#1161]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

Give the ability to use the special object __urn:vcloud:firewallGroup:internal__ in sourceFirewallGroups  and/or destinationFirewallGroups a distributed firewall rule.
urn:vcloud:firewallGroup:internal can be used to specify all internal vDC Group network traffic
(ref: https://developer.vmware.com/apis/vmware-cloud-director/latest/data-structures/FirewallRule/ )

### New or Affected Resource(s)

* vcd_nsxt_distributed_firewall_rule
* vcd_nsxt_distributed_firewall

### References
na

**Comment by JouHouFin** [comment]
Isn't this possible as of now? I just successfully created a vcd_nsxt_distributed_firewall resource with rule having `["urn:vcloud:firewallGroup:internal"] ` in `source_ids` and `destination_ids`.

**Comment by carmine73** [comment]
> Isn't this possible as of now? I just successfully created a vcd_nsxt_distributed_firewall resource with rule having `["urn:vcloud:firewallGroup:internal"] ` in `source_ids` and `destination_ids`.

Correct, I was able to do the same, but I guess there is a limitation on the UI, because "internal" cannot be used:
when I try to edit via UI a rule with `internal` I got this message ` Invalid UUID string: internal`
